packages/http/src/validator/index.ts

```diff
@@ -34,6 +34,22 @@ const checkBodyIsProvided = (requestBody: IHttpOperationRequestBody, body: unkno
     )
   );
 
+const isMediaTypeDefinedInContents = (mediaType?: string, contents?: IMediaTypeContent[]): boolean =>
```

Comment:
I'd call it `isMediaTypeSupportedInContents` otherwise 
```
    O.fromNullable(mediaType),
    O.fold(
      () => true.     // it's not really defined right 
```

Reply:
Done